tests/integration_tests/build/test_build.py

```diff
@@ -59,3 +59,16 @@ def test_build_release_with_features(test_session_root_path):
         '--features "{}"'.format('vsock'),
         '--release'
     )
+
+
+def test_arm_build_release(test_session_root_path):
+    """Test if a release-mode build works for supported features."""
```

Comment:
nit: the comment does not match the test name, it refers to generic supported features.